lib/progressrus.rb

```diff
@@ -21,6 +21,15 @@ def initialize
     end
   end
 
+  class StoreNotFoundError < StandardError
+    def initialize(store)
+      message = <<~MSG
+        The store `#{store}` does not exists
```

Comment:
It might be nice to include available stores in the exception:

`Available stores: #{Progressrus.stores.keys.join(', ')}`

Reply:
Great idea.